### DISCUSSION TITLE
How can I Copy Colors, Field options, etc..

### DISCUSSION THREAD

**Discussion by RiftsSirDrake** [marcusolsson/obsidian-projects#755] [Help]
Hey Folks,

I'm playing around with projects to see the best way to set it up to reduce effort and complexity, but still get the granularity I'm looking for in tracking the dozen projects I've got in flight. I've seen that I can setup the options for fields (like a status field as an example, with in dev, in qa, etc...), but it doesn't seem to carry over when I duplicate the project and change the target folder? Am I doing something wrong here? My goal is to setup my first project, then duplicate that as a template to cutdown on repetitive setup work.

As a related question, I've gone through and setup colors on a board view, can I copy those to another view in the same project? I setup my board first, then started work on a calendar, and I'd love to avoid setting up all the colors a second time if possible!

Thanks!

**Comment by Acylation** [comment]
Yeah that's so painful. This is not yet supported atm but tracked by issue #779.

**Reply by tillmanj** [reply]
It's amazingly painful when you create new projects constantly. It rapidly becomes unreasonable to set up new projects with fields, colors, sorts, and filters. So everything gets lumped into one project folder with lots of weird filter and sort rules to pretend like you have multiple projects instead of just one big list of files.
